gui/app/(auth)/register.tsx

```diff
@@ -25,4 +32,11 @@ const styles = StyleSheet.create({
         alignItems: 'center',
         justifyContent: 'center',
     },
+    body:{
+        width:'100%',
+        height:'100%',
+        backgroundColor:'rgba(52, 52, 52, 0.8)',
```

Comment:
If we're having multiple themes, is this color something that looks visible on all of them?